src/System.Private.ServiceModel/tools/setupfiles/sync-pr.cmd

```diff
@@ -0,0 +1,43 @@
+@echo off 
+setlocal
+
+REM 
+REM This calls a web service, specified on the command line, which syncs a 
+REM remote server's repo to the correct PR. 
+REM This script is meant to be called from a GitHub Pull Request. 
+REM If you must, you can fake it by setting the 'ghprbPullId' environment variable
```

Comment:
I would remove this. I think your previous comment (disclaimer) is good - this is only meant to run on CI (so devs, don't try to run it on your box :))